src/app/model.ts

```diff
@@ -104,6 +110,22 @@ export class Model {
     }
   }
 
+
+
+  public LoadInitialData(selectedDt) {
+    if (selectedDt != null) {
+      this.loadLevels(selectedDt);
+      this.loadDataset(selectedDt, selectedDt.StartDate, 1);
+
+    }
+  }
+
```

Comment:
That's a lot of spaces. I think we need to add some kind of linter to the project to clean up the code.